client/main/tableview.js

```diff
@@ -759,16 +759,39 @@ const TableView = SilkyView.extend({
                     case 'ordinal':
                         if (Number.isInteger(number))
                             value = number;
-                        else if ( ! this.currentColumn.autoMeasure)
-                            throw {
-                                title: 'Integer value required',
-                                message: 'Nominal and Ordinal variables only accept integer values',
-                                type: 'error'
-                            };
+                        else if ( ! this.currentColumn.autoMeasure) {
+                            let found = false;
+
+                            if (typeof value === 'string') {
+                                for (let i = 0; i < this.currentColumn.levels.length; i++) {
+                                    let levelInfo = this.currentColumn.levels[i];
+                                    if (value === levelInfo.label) {
+                                        value = levelInfo.value;
+                                        found = true;
+                                        break;
+                                    }
+                                }
+                            }
+
+                            if (!found) {
+                                throw {
+                                    title: 'Integer value required',
+                                    message: 'Nominal and Ordinal variables only accept integer values',
```

Comment:
as in, if the user enters text that has no value, treat it as a label and request a value using the input message box?